core/store/migrations/migration1609963213/migrate.go

```diff
@@ -0,0 +1,30 @@
+package migration1609963213
+
+import "github.com/jinzhu/gorm"
+
+// Migrate renames eth_request_event_specs to direct_request_specs and adds on_chain_job_spec_id
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+		CREATE TABLE flux_monitor_specs (
+			id SERIAL PRIMARY KEY,
+			contract_address bytea NOT NULL CHECK (octet_length(contract_address) = 20),
+			precision integer,
+			threshold real, 
+			absolute_threshold real,
+			poll_timer_period bigint,
+			poll_timer_disabled boolean,
+			CHECK (poll_timer_disabled OR poll_timer_period > 0),
```

Comment:
doing this instead of our models.Duration / models.Interval as we discussed in team-core

Reply:
Hmm. Could it perhaps be squashed into one field that is nullable?